assemblies/assembly_install-on-jws/main.adoc

```diff
@@ -14,7 +14,7 @@ This document describes how to install {PRODUCT} {PRODUCT_VERSION} on JBoss Web
 
 .Prerequisites
 * You have reviewed the information in {URL_PLANNING_INSTALL}[_{PLANNING_INSTALL}_].
```

Comment:
As a team, we adapted this format for prerequisites. We intentionally use the passive voice here. Because prerequisites are things that you should have done, they need to be in the past tense. We  list it in our[ peer review guide.](https://docs.google.com/document/d/11M6xgFqJI_7fKWs0V5B-Mm03tOzua_05ECJ9BmlHPbk/edit#)

One reason not to use the imperative in prereqs is that they would look an awful lot like the actual procedure.